src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setInputText] = useState('');
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const normalizeQuery = query.trim().toLowerCase();
+    const normalizeTitle = movie.title.toLowerCase();
+    const normalizeDescription = movie.description.toLowerCase();
+
```

Comment:
u can optimize this code make separate function for it

Reply:
agree, let's do it, you just repeat yourself

Reply:
So, why do you ignore it?

Reply:
Make a separate function to check if `str`.includes `substr`
also, you can move normalize string to this function